composer.json

```diff
@@ -22,6 +22,7 @@
     "php": ">=7.2",
     "guzzlehttp/guzzle": "^6.3.3|^7.0",
     "nette/di": "~3.0.0",
+    "nette/http": "^3.1",
```

Comment:
Why is nette/http needed?

Reply:
I used the `\Nette\Http\Url::parseQuery` method for display GET parameter as array

Reply:
Can you use guzzle utils for that or php native functions? Add extra dependency just for tracy bar seems little bit heavy.